src/stream-chat-net/IPermissionService.cs

```diff
@@ -0,0 +1,55 @@
+using System.Threading.Tasks;
+
+namespace StreamChat
+{
+    public interface IPermissionService
```

Comment:
* missing get permission
* also group roles and permissions

Reply:
Added get permission and assignRole. Is there anything else?